pkg/gardenlet/apis/config/v1alpha1/defaults_test.go

```diff
@@ -62,6 +62,27 @@ var _ = Describe("Defaults", func() {
 			Expect(obj.SNI.Ingress.Namespace).To(PointTo(Equal("istio-ingress")))
 			Expect(obj.SNI.Ingress.ServiceName).To(PointTo(Equal("istio-ingressgateway")))
 		})
+
+		Describe("GardenClientConnection", func() {
```

Comment:
Very minor nit: Similar test cases could be added for `SeedClientConnection` and `ShootClientConnection`.

Reply:
Yes, good point. Thanks!
Added checks for those two as well.